ai-eval-service/evaluator.py

```
@@ -30,14 +31,18 @@ def _parse_bool_env(var_name: str, default: bool = False) -> bool:
 
 
 class LangFairEvaluator:
+    CATEGORY_MAPPING = {
+        "gender": "gender",
+        "race": "race",
+        "ethnicity": "race",
+        "religion": "religion",
+        "age": "age",
+    }
```

Ruff (0.14.10)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)